server/background.py

```diff
@@ -0,0 +1,31 @@
+import threading
+import time
+import schedule
+
+
+# Copied from https://schedule.readthedocs.io/en/stable/background-execution.html
+
+
+def run_continuously(interval=1):
+    """Continuously run, while executing pending jobs at each
+    elapsed time interval.
+    @return cease_continuous_run: threading. Event which can
```

Comment:
Could just tack `-> threading.Event` onto the call signature so the IDE will pick this up as well!